src/Microsoft.Build.Sql.Templates/sqlproject/SqlProject1.sqlproj

```diff
@@ -1,9 +1,16 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build">
-  <Sdk Name="Microsoft.Build.Sql" Version="###ASSEMBLY_VERSION###" />
-  <PropertyGroup>
-    <Name>SqlProject1</Name>
-    <DSP>Microsoft.Data.Tools.Schema.Sql.{TargetPlatform}DatabaseSchemaProvider</DSP>
-    <ModelCollation>1033, CI</ModelCollation>
-  </PropertyGroup>
-</Project>
\ No newline at end of file
+<Project Sdk="Microsoft.Build.Sql/###ASSEMBLY_VERSION###" DefaultTargets="Build">
+
+    <PropertyGroup>
+        <SqlTargetName>SqlProject1</SqlTargetName>
```

Comment:
The default for `SqlTargetName` is actually just `$(MSBuildProjectName)` so it does seem like the `Name` is not honored.

Reply:
Yes, that is why I changed it. Changing the name did not change the filename at all. So instead of using a property that literally nothing else uses (`Name`), I bubbled up the existing property from SSDT.